docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+Color: All the labels in this category should use ![#874faf](https://via.placeholder.com/15/874faf/000000?text=+) `#874faf`.
+
+|Name| Description  | Example
+|---|---|------|
+|Help wanted| This is a good issue for a contributor to take on and submit a solution | `Community:Help wanted` |
+
+### Type
+
+Use these labels for providing additional information type of the issue.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Bug| Your classic code defect.| ![#d73a4a](https://via.placeholder.com/15/d73a4a/000000?text=+) `#d73a4a` | `Type:Bug` |
+|Feature request| This is a new feature or an enhancement to an existing one.| ![#8f31ed](https://via.placeholder.com/15/8f31ed/000000?text=+) `#8f31ed` | `Type:Feature request` |
+|Customer ask| An enhancement or feature requested by a customer (use this label if the feature request was originated by a customer).| ![#8f31ed](https://via.placeholder.com/15/8f31ed/000000?text=+) `#8f31ed` | `Type:Customer ask` |
+|Parity| The issue describes a gap in parity between two or more platforms.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Parity` |
+|Technical debt| The issue involves refactoring existing code to make it easier to maintain, follow best practices, improved test coverage, etc.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Technical debt` |
+|Team agility| An issue targeted to reduce friction to releasing new versions of the SDKs.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Team agility` |
+
+### Size
+
+Use these to assign an estimated level of effort to resolve an issue and assist with the estimation process.
+
+Color: All the labels in this category should use ![#91e3ea](https://via.placeholder.com/15/91e3ea/000000?text=+) `#91e3ea`.
+
+|Name| Description  | Example
+|---|---|------|
+|Small| The issue is simple and well understood, it will take a day or less to complete | `Size:S` |
+|Medium| The issue is not very complex and it is well understood, it will take 1 to 3 days to complete | `Size:M` |
+|Large| The issue complex but it is well understood, it will take 4 to 8 days to complete | `Size:L` |
+|Extra Large| The issue very complex or not very well defined, it will take 9 to 14 days to complete. In this case, it is probably better to rethink the issue and break it down in smaller tasks | `Size:XL` |
+
+### Automation
+
+These labels are applied to PRs and used to trigger or disable github workflows.
+
+Color: All the labels in this category should use ![#cccccc](https://via.placeholder.com/15/cccccc/000000?text=+) `#cccccc`.
+
+|Name| Description  | Example
+|---|---|------|
+|no parity| PR does not need to be applied to other languages. **Note:** if you don't apply the `No parity` to a dotnet PR, the automation workflow will generate parity issues in Python, JS and Java.  | `Automation:No parity` |
```

Comment:
Should we do the automation around by labelling it "Parity needed" and the bot pick it up and spread it across platforms?

Reply:
I'll fine tune once we define how we handle this, in the SDK world parity needed should be the default, you would use automation to ignore parity or to specify that you need parity for just one platform but again, I'll address this once we spec out automation as part of a separate effort.